src/utils/helpers.js

```diff
@@ -52,6 +52,19 @@ const _ = {
     }
 
     return input;
+  },
+  uniq: function(array) {
+    const seen = {};
+    let uniques = [];
+
+    for (let i = 0; i < array.length; i++) {
+      const element = array[i];
+      if (seen[element] !== element) {
+        seen[element] = element;
+        uniques = uniques.concat(element);
+      }
+    }
+    return uniques;
```

Comment:
Imho there is no point in copying the accumulator each time.
But the Object.keys works, I'll take that :)